sources/common/inc/Timestamp.h

```diff
@@ -0,0 +1,45 @@
+/// @file common/inc/Timestamp.h
+///
+/// The signal timestamp class.
+///
+/// @par Full Description
+/// The signal time stamp class.
+///
+/// @ingroup Time
+///
+/// @par Copyright (c) 2017 vcdMaker team
+///
+/// Permission is hereby granted, free of charge, to any person obtaining a
+/// copy of this software and associated documentation files (the "Software"),
+/// to deal in the Software without restriction, including without limitation
+/// the rights to use, copy, modify, merge, publish, distribute, sublicense,
+/// and/or sell copies of the Software, and to permit persons to whom the
+/// Software is furnished to do so, subject to the following conditions:
+///
+/// The above copyright notice and this permission notice shall be included
+/// in all copies or substantial portions of the Software.
+///
+/// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+/// OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+/// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+/// THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+/// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+/// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+/// IN THE SOFTWARE.
+
+#pragma once
+#include "SafeUInt.h"
+
+/// @defgroup Time Time
+///
+/// @brief The group defines entities for time operations.
+///
+/// @par Full Description
+/// The group defines the time stamp class. It also provides means to make time
+/// unit conversions.
+
+namespace TIME
```

Comment:
Dowe really need another namespace for just one class? This class is connected with `SIGNAL`.

Reply:
There will be more moved into from the utils.